recipes/xtr/all/conanfile.py

```diff
@@ -17,19 +17,38 @@ class XtrConan(ConanFile):
         "fPIC": [True, False],
         "enable_exceptions": [True, False],
         "enable_lto": [True, False],
+        "enable_io_uring": [True, False],
+        "enable_io_uring_sqpoll": [True, False],
+        "sink_capacity_kb": ["default"] + list(map(lambda x : 2**x, range(2, 15))) # 4k to 16MB
     }
     default_options = {
         "fPIC": True,
         "enable_exceptions": True,
         "enable_lto": False,
+        "enable_io_uring": True,
+        "enable_io_uring_sqpoll": False,
+        "sink_capacity_kb": "default"
     }
+
     generators = "make"
 
+    def config_options(self):
+        if tools.Version(self.version) < "1.0.1":
+            del self.options.sink_capacity_kb
+        if tools.Version(self.version) < "2.0.0":
+            del self.options.enable_io_uring
+            del self.options.enable_io_uring_sqpoll
+
+    def get_option(self, name):
+        if not name in self.options:
+            return None
+        return getattr(self.options, name)
+
```

Comment:
```suggestion
```
Not needed. Use `self.options.get_safe()` instead.

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#options